server/api.js

```diff
@@ -1,9 +1,32 @@
 import { Router } from "express";
+import pool from "./db";
 
 const router = Router();
 
 router.get("/", (_, res) => {
 	res.json({ message: "Hello, world!" });
 });
 
+router.get("/users", async (req, res) => {
+  try {
+    const users = await pool.query("SELECT * FROM users");
```

Comment:
Also the ``*`` here could change to specific columns

Reply:
Do we need this route for the ticket? May be we can remove this GET endpoint ("/user)?